src/main/groovy/life/qbic/portal/sampletracking/web/presenters/SampleImportPresenter.groovy

```diff
@@ -0,0 +1,40 @@
+package life.qbic.portal.sampletracking.web.presenters
+
+import life.qbic.datamodel.samples.Sample
+import life.qbic.portal.sampletracking.trackinginformation.list.ModifySampleListOutput
+import life.qbic.portal.sampletracking.web.ViewModel
+
+/**
+ * <add class description here>
+ *
+ * @author: Sven Fillinger
+ *
+ */
+class SampleImportPresenter implements ModifySampleListOutput{
+
+    final ViewModel viewModel
+
+    SampleImportPresenter(ViewModel viewModel) {
+        this.viewModel = Objects.requireNonNull(viewModel, "ViewModel must not be null")
+    }
+
+    @Override
+    void addSamples(List<Sample> samples) {
+        viewModel.samples << samples
```

Comment:
Why use the method pointer / shift operator here?
Did you mean 
```suggestion
        viewModel.samples <<= samples
```
I would suggest to keep that more readable to use the `List` method `addAll()`